database/migrations/2020_06_02_204137_add_group_type_id_to_campaigns_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddGroupTypeIdToCampaignsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->unsignedInteger('group_type_id')->nullable()->index()->comment('The group type id the group is for.');
+            $table->foreign('group_type_id')->references('id')->on('group_types');
```

Comment:
It wouldn't hurt to an index on the `group_type_id` field we're adding here, as we'll eventually want to filter a `GET /campaigns` index query by `group_type_id`.

We should just be able to additionally add:
```
$table->index('group_type_id');
```

Reply:
This shouldn't block merging though, could make sense once we add to the `indexes` array in the `Campaign` model.

Reply:
👍  I might add it in on this PR just to keep track of it

Reply:
Doesn't this line already make it an index with `->index()`?

Reply:
That's what i thought as well @katiecrane

Reply:
I was wondering if i then also need to add it to the index array on the model

Reply:
Ooooo that array looks to be out of date as we have more that just `id` indexed on this table

Reply:
Argh I totally missed the `index` call in thsi line! 🤦‍♂️ I think it'd be fine to hold off on modifying the campaign `indexes` in a separate PR. sorry for the false alarm!